float64[1] u
